src/consumer/component/link-generator.js

```diff
@@ -49,16 +48,20 @@ function _getLinkContent(filePath: string, importSpecifier?: Object): string {
         // @todo: implement
       }
     }
+    if (filePath.endsWith('.st.css')) {
+      // Wix Stylable format
+      return ':import { -st-from: "{filePath}.css"; }';
```

Comment:
Because:
1) there is a chance we'll need the importSpecifiers same as we have for .js and .ts (see their syntax --st-from and --st-named).
2) this part is going to be a separated plugin, so it's easier to cut and paste it somewhere else this way.